extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -32,8 +33,8 @@ class UpdateReplicationStatus extends BackbeatTask {
     }
 
     _createCredentials(authConfig, roleArn, log) {
-        if (authConfig.type === 'account') {
-            return new AccountCredentials(authConfig, log);
+        if (authConfig.type === 'service') {
```

Comment:
IMO it can be useful to keep the 'account' type for very simple deployments or testing.